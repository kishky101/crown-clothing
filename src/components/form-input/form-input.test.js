import renderer from 'react-test-renderer';
import FormInput from './form-input.component';

test('creating form-input snapshot', () => {
    const forminput = renderer.create(<FormInput />).toJSON
    expect(forminput).toMatchSnapshot();
})