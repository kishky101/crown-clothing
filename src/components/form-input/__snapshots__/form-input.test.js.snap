// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creating form-input snapshot 1`] = `[Function]`;
